schema/schemaspec/spec.go

```diff
@@ -117,7 +117,25 @@ func (a *Attr) Strings() ([]string, error) {
 	for _, item := range lst.V {
 		sv, err := StrVal(item)
 		if err != nil {
-			return nil, fmt.Errorf("schema: failed parsing item %q to string: %w", item, err)
+			return nil, fmt.Errorf("schemaspec: failed parsing item %q to string: %w", item, err)
+		}
+		out = append(out, sv)
+	}
+	return out, nil
+}
+
+// Bools returns a slice of bools from the Value of the Attr. If The value is not a ListValue or its
+// values cannot be converted to bools an error is returned.
+func (a *Attr) Bools() ([]bool, error) {
+	lst, ok := a.V.(*ListValue)
+	if !ok {
+		return nil, fmt.Errorf("schema: attribute %q is not a list", a.K)
+	}
+	out := make([]bool, 0, len(lst.V))
+	for _, item := range lst.V {
+		sv, err := BoolVal(item)
+		if err != nil {
+			return nil, fmt.Errorf("schemaspec: failed parsing item %q to bool: %w", item, err)
```

Comment:
Error look weird if you wrap the first error in BoolVal. I think change the last error in BoolVal and remove this wrapping.